pkg/execution/execute.go

```diff
@@ -136,7 +136,7 @@ func ExecuteRulesets(rulesets []windup.Ruleset, baseLocation, datadir string) (s
 	for _, ruleset := range rulesets {
 		sourceFiles = append(sourceFiles, ruleset.SourceFile)
 	}
-	conversion.ConvertWindupRulesetsToAnalyzer(rulesets, baseLocation, filepath.Join(dir, "rules"))
+	conversion.ConvertWindupRulesetsToAnalyzer(false, rulesets, baseLocation, filepath.Join(dir, "rules"))
```

Comment:
Does this mean the flattening isn't actually tested?

Reply:
@fabianvf good point, will add to the test and see if anything breaks